pygsti/models/explicitmodel.py

```diff
@@ -1521,11 +1521,11 @@ def extract_unitary(Umx, U_sslbls, extracted_sslbls):
             return U_extracted
 
         def add_availability(opkey, op):
-            if opkey == _Label(()) or opkey.is_simple():
+            if opkey == _Label(()) or opkey.is_simple:
```

Comment:
Consider swapping the order of these two boolean checks. The ``is_simple`` check will be faster and might be true more often anyway.

Reply:
Great observation!